app/web/features/communities/discussions/DiscussionsSection.tsx

```diff
@@ -47,18 +47,19 @@ export default function DiscussionsSection({
       {discussionsError && (
         <Alert severity="error">{discussionsError.message}</Alert>
       )}
-      <Button
-        size="small"
-        className={classes.createResourceButton}
-        component={Link}
-        to={`${routeToCommunity(
+      <Link
+        href={`${routeToCommunity(
           community.communityId,
           community.slug,
           "discussions"
         )}#${composingDiscussionHash}`}
       >
-        {NEW_POST_LABEL}
-      </Button>
+        <a>
+          <Button size="small" className={classes.createResourceButton}>
+            {NEW_POST_LABEL}
+          </Button>
+        </a>
```

Comment:
```suggestion
          <Button size="small" className={classes.createResourceButton} component="a">
            {NEW_POST_LABEL}
          </Button>
```
So the HTML is valid

Reply:
Quite right. I note in this case I will need to add `passHref` to the `Link`, which is necessary when the child is not "a"